services/nvidia-gpu-operator/23.3.1/defaults/cm.yaml

```diff
@@ -35,4 +35,4 @@ data:
       # this can cause a mismatch in cuda versions
       # see: https://github.com/NVIDIA/gpu-operator/issues/389#issuecomment-1215781064
       # for more details
-      version: v22.9.1
+      version: v23.3.1
```

Comment:
@faiq not sure about this but I bumped the image here

Reply:
I think we can get rid of this value. We needed it for the bug that was described, but it's safe to remove for now.

Reply:
removed in https://github.com/mesosphere/kommander-applications/pull/1156/commits/3f37e43efa7257a13f242f9c6fc393fa4068bd8d

Reply:
Just a question, if we wanted to automate these new values, how should we do it? 

One thought I had was we could potentially put all the values in this override config map and parse the configmap, and update the versions in the `hack/airgapped/missing-images.json` file. 

If we did that then we do need it.

What do you think of this approach, do you have others in mind?